src/components/ArgoCDDetailsWidget.tsx

```diff
@@ -159,10 +109,10 @@ export const ArgoCDDetailsWidget = ({ entity }: { entity: Entity }) => {
 };
 
 const ArgoCDDetails = ({ entity }: { entity: Entity }) => {
-  const appName = useArgoCDAppData({ entity });
+  const { appName, appSelector } = useArgoCDAppData({ entity });
 
-  const { loading, value, error } = useAppDetails({
-    appName,
+  const { loading, value, error } = !!appName ? useAppDetails({ appName }) : useListAppDetails({

```

Comment:
It's illegal to invoke hooks conditionally. I suggest moving this switch into the hook useAppDetails

Reply:
Resolved in c0f8821f1ca554966473c7be6e04a9b4c6df9f7f.